_layouts/default.html

```diff
@@ -56,6 +56,7 @@
                     <li><a href="{{ root_path }}#quick-start">Quick Start</a></li>
                     <li><a href="{{ root_path }}#download">Download</a></li>
                     <li><a href="//formats.kaitai.io/">Format Gallery</a></li>
+                    <li{% if page.url == '/news/' %} class="active"{% endif %}><a href="/news">News</a></li>
```

Comment:
I doubt `page.url == '/news/'` will work when you're not directly at https://kaitai.io/news/, but on a specific subpage like http://kaitai.io/news/2022/07/08/kaitai-struct-v0.10-released.html.

Reply:
Correct, that does not work. The options I can find are:

```liquid
<li{% if page.url contains '/news/' %} class="active"{% endif %}> ...
```

or

```liquid
{% assign substring = page.url | slice: 0, 6 %}
<li{% if substring == '/news/' %} class="active"{% endif %}> ...
```